docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md

```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
 
 Visual C++ manages a single .RC resource file and a corresponding .H header file as a tightly coupled pair of files. When you edit and save resources in an .RC file, you indirectly edit and save symbols in the corresponding .H file. Although you can open and edit multiple .RC files at a time (using Visual C++'s MDI user interface) for any given .RC file you indirectly edit exactly one corresponding header file.
 
-### Symbol Header File
+### Resource View's Resource Includes dialog
 
-By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). Using the **Resource Includes** command from the **View** menu in Visual C++, you can change the name of this header file by updating the Symbol Header File file in the **Set Includes** dialog box.
+#### Symbol Header File
 
-### Read-Only Symbol Directives
+By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). The **Symbol Header File:** section in the **Resource Includes** dialog in Visual C++, lets you change the name of this header file. Enter a new file name in the section's edit box.
```

Comment:
Let's add that the Resource includes is accessible from right clicking the .rc file form the resource view

Reply:
Should `#include`d be back ticked?
[NOTE!] needs to be properly formatted. I don't know what that should be.
I added line #51 which addresses AnjuDel's comment. I included Resource View because Resource Includes can be accessed in at least three different ways: from the .rc file in Solution Explorer, from the VS ToolBar->View->Other Windows, and through accelerator keys. Not wanting to risk being too tedious, I left out discussion of Open With and the subsequent choice of viewing editor (JSON? XML? etc).

Reply:
When #include is part of code, it should have back ticks. When it is part of UI, e.g. **Resource Includes**, no.
The format for notes is, and I'll code escape this so the GitHub formatter doesn't mangle it:
`> [!NOTE]
> Note text starts here`

Reply:
I would like to re-add my note about relative path-names since it is valid and necessary for the `RC` Compiler to find the header files:

> [!NOTE]
If the header file is not in the same directory as the project, pre-pend the new `MYRESOURCE.H` header name in the **Symbol Header File** edit box with a relative path.

Reply:
Minor edits: "If the header file isn't in the same directory as the project, in **Symbol Header File**, prepend the new `MYRESOURCE.H` header name with a relative path.